templates/ts-react-apollo-client/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
This won't be needed once the templates are deployed to a user's machine through our CLI, it is only there for working around the workspaces issue right? I did not add it for this reason, as the user is getting environment variables which they don't need.

`SKIP_PREFLIGHT_CHECK=true yarn start`

Reply:
I see this actually happening when initializing the project using graphql-cli.
Not sure about our CLI.